src/models.cxx

```diff
@@ -1083,9 +1083,12 @@ int SmallStrainCreepPlasticity::RJ(const double * const x, TrialState * ts,
   u_n = 0.0;
   p_n = 0.0;
 
+  double * hist = (h_np1.empty() ? nullptr : &h_np1[0]);
+  double * hist_tss = (tss->h_n.empty() == 0 ? nullptr : &(tss->h_n[0]));
```

Comment:
`tss->h_n.empty()`

not

`tss->h_n.empty() == 0`

Reply:
Doh! Fixed.